capstone/src/main/java/com/google/sps/servlets/Announcement.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Announcement {
+  private String author;
+  private String club;
+  private long time;
+  private String content;
+
+  public Announcement(Entity entity) {
+    this.author = entity.getProperty("author").toString();
+    this.club = entity.getProperty("club").toString();
+    this.content = entity.getProperty("content").toString();
+    this.time = Long.parseLong(entity.getProperty("time").toString());
```

Comment:
We shouldn't include Entity dependencies in this class since Entities are a DataStore-specific concept (whereas announcements are just general objects). So it's actually better to longhand and include each individual field in the constructor and in the servlet, manually call new Announcement(entity.getProperty("author"), entity.getProperty("club"), ...)

Reply:
I see, that makes sense